tests/test_images.py

```diff
@@ -6,12 +8,26 @@
 
 @pytest.mark.asyncio
 async def test_build_from_remote_file(docker):
-    remote = ("https://raw.githubusercontent.com/"
-              "komljen/dockerfile-examples/master/redis/Dockerfile")
+    remote = ("https://gist.githubusercontent.com/barrachri/"
+              "16a735971379352635a92d65036145f1/raw"
+              "/2b814ad3bf24af94a5623df03678816e37f2b6b7/Dockerfile")
     name = "image:1.0"
     await docker.images.build(tag=name, remote=remote)
-    image = await docker.images.get(name)
-    assert image
+    # build the image requires time
+    # we hold on for max 10 secs
+    # until we are sure the image
+    # is available locally using `get`
+    try:
+        with aiohttp.Timeout(10):
+            while True:
+                try:
+                    image = await docker.images.get(name)
```

Comment:
Should we do something like this https://github.com/aio-libs/aiodocker/blob/3b91a65ae2ddfc5aa6e2c2198b88250eb04339fd/tests/test_integration.py#L114-L119  here? Wait for log line, and then check `docker.images` otherwise dump log in case of timeout.